docs/rst/notes/notes.rst

```diff
@@ -30,28 +34,31 @@ This release includes the following **dependencies update**:
         - New Version
     *   - Foonathan Memory Vendor
         - `eProsima/foonathan_memory_vendor <https://github.com/eProsima/foonathan_memory_vendor>`_
-        - `v1.3.0 <https://github.com/eProsima/foonathan_memory_vendor/releases/tag/v1.3.0>`_
+        - `v1.3.1 <https://github.com/eProsima/foonathan_memory_vendor/releases/tag/v1.3.1>`_
         - `v1.3.1 <https://github.com/eProsima/foonathan_memory_vendor/releases/tag/v1.3.1>`_
     *   - Fast CDR
         - `eProsima/Fast-CDR <https://github.com/eProsima/Fast-CDR>`_
-        - `v1.0.27 <https://github.com/eProsima/Fast-CDR/releases/tag/v1.0.27>`_
         - `v1.1.0 <https://github.com/eProsima/Fast-CDR/releases/tag/v1.1.0>`_
+        - `v2.1.1 <https://github.com/eProsima/Fast-CDR/releases/tag/v2.1.1>`_
```

Comment:
I think we should target [Fast CDR v2.1.2](https://github.com/eProsima/Fast-CDR/releases/tag/v2.1.2). Update this everywhere